.vscode/settings.json

```diff
@@ -43,6 +43,25 @@
     "TZ": "UTC"
   },
 
+  // These custom rules are read in extensions/ql-vscode/.markdownlint-cli2.cjs
+  // but markdownlint only considers that config when linting files in
+  // extensions/ql-vscode/ or its subfolders. Therefore, we have to explicitly
+  // load the custom rules here too.
+  "markdownlint.customRules": [
+    "./extensions/ql-vscode/node_modules/@github/markdownlint-github/src/rules/no-default-alt-text.js",
+    "./extensions/ql-vscode/node_modules/@github/markdownlint-github/src/rules/no-generic-link-text.js"
+  ],
+
+  // This ensures that the rule enablement done by github-markdownlint is considered
+  // by the extension too.
+  //
+  // Additional nonfiguration of the default rules is done in .markdownlint.json,
+  // which is picked up by the extension automatically, and read explicitly in
+  // .markdownlint-cli2.cjs
+  "markdownlint.config": {
+    "extends": "./extensions/ql-vscode/node_modules/@github/markdownlint-github/style/accessibility.json"
```

Comment:
Should this also include `./extensions/ql-vscode/node_modules/@github/markdownlint-github/style/base.json`? That also seems to be included in the default `@github/markdownlint-github` configuration.